sshclient.go

```diff
@@ -106,6 +106,11 @@ func (c *Client) Close() error {
 	return c.client.Close()
 }
 
+// UnderlyingClient get the underlying client network connection.
```

Comment:
这里应该改为 get the underlying client。因为 ssh.Client 不是一个网络连接，它包含很多东西。
This should be changed to `get the underlying client`. Because `ssh.Client` isn't a network connection, it contains many things.

Reply:
Thanks 没注意如此多的细节 给您带来麻烦了  下次会注意